File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/org/openapitools/client/model/Inline_object_1.scala

package org.openapitools.client.model


case class Inline_object_1 (
    _model: Option[SkinModel],
    /* The URL which Mojang servers will download and apply the skin */
    _url: String
)
object Inline_object_1 {
    def toStringBody(var_model: Object, var_url: Object) =
        s"""
        | {
        | "model":$var_model,"url":$var_url
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-api/src/main/kotlin/org/openapitools/model/InlineObject1.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.SkinModel
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Request Mojang to download a skin from an URL and apply to the player
 * @param model 
 * @param url The URL which Mojang servers will download and apply the skin
 */
data class InlineObject1 (

        @get:NotNull 
        @JsonProperty("url") val url: String,

        @JsonProperty("model") val model: SkinModel? = null
) {

}

